Source/santad/main.mm

```diff
@@ -85,6 +87,52 @@ static void SantaWatchdog(void *context) {
   }
 }
 
+static void FinishInstall() {
+  // Wait for com.google.santa.daemon to be removed.
+  while (1) {
+    // Record if com.google.santa.daemon is an active and enabled system extension.
+    BOOL google_santa_active_enabled = NO;
+    NSDictionary *system_extensions =
+      [NSDictionary dictionaryWithContentsOfFile:@"/Library/SystemExtensions/db.plist"];
```

Comment:
I think I have concerns about stability here. Would something like `propertiesRequestForExtension:queue:` work here? https://developer.apple.com/documentation/systemextensions/ossystemextensionrequest/propertiesrequest(forextensionwithidentifier:queue:)?language=objc

Reply:
Looks like OSQuery's also using it as well https://github.com/osquery/osquery/issues/6738

https://github.com/osquery/osquery/blob/8920f3343613b8a7a9521d740af5e6e4650ee848/osquery/tables/system/darwin/system_extensions.mm#L29

Reply:
We ran some experiments here. Unfortunately that API only returns properties for extensions with the same team ID